MIGRATION.md

```diff
@@ -1,3 +1,10 @@
+ # Upgrading from 7.18.0 to 7.19.0
```

Comment:
```suggestion
 # Upgrading from 7.x to 7.19.x
```

What about this? Not sure if we should do it this way.

Reply:
Yeah, I was also not clear on that... 

I think I would maybe instead add an overall section "Deprecations in 7.x" and collect this there, as it is not really that relevant that this is in 7.19!

Reply:
yeah that sounds fine to me!

Reply:
I added a deprecation section to migrations, I also added everything from changelog I could find since 7.0.0!